acq4/modules/MultiPatch/multipatchTemplate.py

```diff
@@ -76,9 +76,6 @@ def setupUi(self, MultiPatch):
         self.slowBtn.setCheckable(True)
```

Comment:
should be fixed in [newest commit](https://github.com/aiephys/acq4/pull/27/commits/92eee0c34eb26a7b9cbaad2ac8f96b8d18d02d4d)